libModule2/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
Is there a particular reason to have the proguard rules file?

Reply:
No particular reason, can be removed if we wish to.

Reply:
I'm fine with leaving this since android library modules usually have it.